maintainers/scripts/haskell/sweep-packages.nix

```diff
@@ -0,0 +1,83 @@
+# Utils for sweeping through packages based on some property discoverable at eval time
+# Useful for narrowing down package candidates on which to test some build time property
+#
+# Example 1: finding packages with patches that no longer apply
```

Comment:
Arguably, here the better approach is to build `srcOnly pkg` and see whether it fails which is much cheaper.